build/replace/Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+pipeline {
+  agent {
+    dockerfile {
+      dir 'build'
+    }
+  }
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '10'))
+    disableConcurrentBuilds()
+  }
+
+  triggers {
+    cron '@midnight'
+  }
+  
+  parameters {
+    booleanParam name: 'dryRun', defaultValue: true, description: 'whether the build should push the changes or not'
+    string(name: 'sedRegexp', defaultValue: '', description: 'The sed replacement: s#search#replace#g')
+    string(name: 'fileSelector', defaultValue: '', description: 'Bash file selector e.g. folder/**/File.*')
+    string(name: 'sourceBranch', defaultValue: 'master', description: 'The branch to apply this change on e.g. master, release/11.1, release/10.0')
+    string(name: 'newBranch', defaultValue: 'replace', description: 'The name of the new branch, which gets pushed if it is not a dry run')
+    string(name: 'commitMessage', defaultValue: '', description: 'The commit message')
```

Comment:
not sure how much value it brings to have these two parameters changeable... IMO it is fine if they are autogenerated by the job, like it is on the other pipelines. But ok for me

Reply:
Normally it has a good reason why we search replace something. Only the sed search replace value can look very complicated, not sure how you want to auto generate it :thinking: